code/modules/fishing/sources/subtypes/surgery.dm

```diff
@@ -25,14 +25,12 @@
 
 	var/mob/living/carbon/carbon = fishing_spot
 	var/list/possible_organs = list()
-	for(var/datum/surgery/organ_manipulation/operation in carbon.surgeries)
-		var/datum/surgery_step/manipulate_organs/manip_step = GLOB.surgery_steps[operation.steps[operation.status]]
-		if(!istype(manip_step))
+	for(var/obj/item/organ/fishable as anything in carbon.organs)
+		if(fishable.organ_flags & (ORGAN_UNREMOVABLE|ORGAN_EXTERNAL|ORGAN_VITAL))
```

Comment:
Why vital? Fishing out brains is kinda funny and I don't see why we'd need such safeguards here

Reply:
Mmm fair enough I guess